js/pulse.js

```diff
@@ -616,6 +751,10 @@
             }, this);
 
             var _onPlayStarted = function() {
+                //Hide a pause ad is there was any
+                if(this._currentPauseAd){
+                    amc.notifyNonlinearAdEnded(this_)
+                }
```

Comment:
Oh good catch, I'll just comment out this code for now as support for pause ads will need some work on the skin.